src/form.tsx

```diff
@@ -119,7 +119,7 @@ const Form: React.FC<FormProps> = ({
         }
 
         setFormErrors(errors);
-        if (onValidationErrors) {
+        if (onValidationErrors && Object.keys(errors).length > 0) {
```

Comment:
unrelated fix, Form was always calling onValidationErrors on submit, it should only be called if some error exists